src/components/button/Button.scss

```diff
@@ -0,0 +1,23 @@
+@use "../../styles/colors" as c;
```

Comment:
you can use `@use "../../styles/colors" as *;` instead of `as c`
that way color variables can can be used by using just $color-grey